public/components/CampaignPerformanceOverview/CampaignPerformanceOverview.js

```diff
@@ -1,4 +1,25 @@
 import React, { PropTypes } from 'react';
+import R from 'ramda';
```

Comment:
How is Ramda used here?

Reply:
Oh I see.

Reply:
`R.isEmpty` to check if an object is empty.